pkg/products/rhsso/reconciler.go

```diff
@@ -202,6 +203,32 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 	return integreatlyv1alpha1.PhaseCompleted, nil
 }
 
+func (r *Reconciler) reconcileCloudResources(ctx context.Context, installation *integreatlyv1alpha1.RHMI, serverClient k8sclient.Client) (integreatlyv1alpha1.StatusPhase, error) {
```

Comment:
I don't understand, this is a totally new function but then I don't see it being called anywhere. How is it used?

Reply:
Good spot. This function should have been removed as part of the rebase on master. there's a rhssocommon/reconciler.go where this should be now. I'll fix it up now.

Reply:
should be fixed now. 👍